ugc/api.cpp

```diff
@@ -37,73 +32,67 @@ void Api::SetUGCUpdate(FeatureID const & id, UGCUpdate const & ugc)
 // static
 UGC Api::MakeTestUGC1(Time now)
 {
-  Rating rating;
-  rating.m_ratings.emplace_back("food" /* key */, 4.0 /* value */);
-  rating.m_ratings.emplace_back("service" /* key */, 5.0 /* value */);
-  rating.m_ratings.emplace_back("music" /* key */, 5.0 /* value */);
-  rating.m_aggValue = 4.5;
+  RatingRecords records;
```

Comment:
How about to rename RatingRecords to Ratings?

Reply:
I think this name will be more clear than this one. But there is already a structure with name Ratings which is not used now. I think it's better to delete Ratings and rename RatingRecords to Ratings or RatingNotes maybe